pkgs/tools/virtualization/amazon-ecs-cli/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl}:
+
+stdenv.mkDerivation rec {
+  name = "amazon-ecs-cli";
+  version = "1.12.1";
+
+  src = fetchurl {
+    url = "https://s3.amazonaws.com/amazon-ecs-cli/ecs-cli-linux-amd64-v${version}";
+    sha256 = "100iv4cchnxl1s02higga5v3hvawi4c7sqva97x34qigr4r7fxwm";
+  };
+
+  unpackCmd = "mkdir bin; cp $src ./bin/ecs-cli";
```

Comment:
Seems to me that `unpackCmd` and `installPhase` can be replaced with
```
install -D $src $out/bin/ecs-cli
```

Reply:
I have not found a way to disable the unpack phase, could you point me in the right direction?

Reply:
A simple way to disable specific phases is to set them to ":", as in
```nix
{
   unpackPhase = ":";
}
```

Reply:
like this?

Reply:
LGTM.  Using `install` would shorten it further, as noted above. Even simpler would be
```nix
runCommand "foo" { src = fetchurl { /* ... */ }; } ''install -D $src $out/bin/foo''
```
There may even be a suitable canned builder that captures this pattern entirely. That said I think what you have is acceptable as is ...